sqldelight-gradle-plugin/src/test/fixtures/javadoc/src/main/sqldelight/com/sample/Test.sq

```diff
@@ -0,0 +1,10 @@
+CREATE TABLE test (
+  _id INTEGER NOT NULL PRIMARY KEY AUTOINCREMENT
+);
+
+/**
+ Javadoc comment yo.
```

Comment:
No. They would have to be stripped from the comment body itself before passing to JavaPoet just as newlines.